documentation/source/users/rmg/input.rst

```diff
@@ -481,6 +481,35 @@ For example ::
 		toleranceMoveSurfaceReactionToCore=5.0,
 	)
 
+Advanced Setting:  Dynamics Criterion Weights
+---------------------------------------------
+In reaction networks reactions that change the number of radicals tend to be much more important
+than other reactions.  Also in many cases specific species within a model are of interest.  
+
+In these sorts of situations RMG implements two different types of weights:
+Reaction Weights: multiplied by the dynamics number for a given reaction at the end of calculations
```

Comment:
Bold `Reaction Weights`.

Also add an extra line so that `Species Weights` appears on a separate line in the html file.